### ISSUE TITLE
UI says searches are regex, but are string literal searches instead

### ISSUE THREAD

**Issue by spinkham** [hvqzao/burp-flow#3]
The extension uses "Pattern.quote" to turn the search input into a quoted string, which java regex library treats as a string literal. Because of this, searches are for strings, and searching by regex is impossible.

Ideally would have a toggle to switch between string literal and regex like the Burp proxy history view, but should at least remove the reference to regex in the UI to limit confusion.

```
if (flowFilterSearchCaseSensitive.isSelected()) {
                        found = Pattern.compile(Pattern.quote(text)).matcher(req).find() || Pattern.compile(Pattern.quote(text)).matcher(resp).find();
                    } else {
                        found = Pattern.compile(Pattern.quote(text), Pattern.CASE_INSENSITIVE).matcher(req).find() || Pattern.compile(Pattern.quote(text), Pattern.CASE_INSENSITIVE).matcher(resp).find();
                    }
```

**Comment by hvqzao** [comment]
Hi, I've updated field description (removed note about regex). If you wish, you can work on this feature and  provide me with proposed diff / make a pull request. Newest version is here: https://github.com/hvqzao/burp-flow/releases/tag/1.15
I will test it for few days and then submit to Bapps. Please let me know if you want to work on it. Thanks!

**Comment by hvqzao** [comment]
I've implemented regex filtering in version 1.16, it is available here: https://github.com/hvqzao/burp-flow/releases/tag/1.16
